src/Test/Perf/util/TraceManager_util.csx

```diff
@@ -0,0 +1,231 @@
+#load "ScenarioGenerator_util.csx"
+
+using System;
+using System.Diagnostics;
+using System.IO;
+
+interface ITraceManager
+{
+    int Iterations { get; }
```

Comment:
The manager, which looks over the running of all the testcase, is a good candidate to mention the iteration. you have a different idea here?